lib/potentialbuild.rb

```diff
@@ -84,6 +84,11 @@ def initialize(client, token, repository, tag_name, commit_sha, branch_name, aut
     @valgrind_counters_results = nil
     @perf_counters_results = nil
     @file_sizes = nil
+    @pr_num_for_comment = nil
```

Comment:
Creates a nil member variable to store the PR number that should be used ONLY for posting the GitHub comment.